lib/driver.js

```diff
@@ -172,11 +171,11 @@ class XCUITestDriver extends BaseDriver {
 
     if (this.opts.browserName) {
       log.info('Safari test requested');
-      this.safari = true;      
+      this.safari = true;
       this.opts.app = undefined;
       this.opts.processArguments = this.opts.processArguments || {};
       if (this.isRealDevice()) {
-        this.opts.bundleId = SAFARI_LAUNCHER_BUNDLE;
+        this.opts.bundleId = SAFARI_BUNDLE_ID;
         this._currentUrl = this.opts.safariInitialUrl || `http://appium.io`;
         this.opts.processArguments.args = [this._currentUrl];
```

Comment:
If this is going straight to Safari, these process arguments need to be changed, I think.